apis/ec2/v1beta1/natgateway_types.go

```diff
@@ -57,8 +57,7 @@ type NATGatewayParameters struct {
 	SubnetIDSelector *xpv1.Selector `json:"subnetIdSelector,omitempty"`
 
 	// Tags represents to current ec2 tags.
-	// +optional
-	Tags []Tag `json:"tags,omitempty"`
+	Tags []Tag `json:"tags"`
```

Comment:
Nit: Should we update the description to explicitly mention that at least one tag appear to be required by the API.